src/mgr/MgrPyModule.cc

```diff
@@ -171,32 +171,54 @@ int MgrPyModule::load()
   }
 
   // Find the class
-  // TODO: let them call it what they want instead of just 'Module'
-  auto pClass = PyObject_GetAttrString(pModule, (const char*)"Module");
-  Py_DECREF(pModule);
-  if (pClass == nullptr) {
-    derr << "Class not found in module '" << module_name << "'" << dendl;
+  PyObject *mgr_module_name = PyString_FromString("mgr_module");
```

Comment:
@bhavishyagopesh why don't you just use `PyImport_ImportModule()` ?


see. https://github.com/ceph/ceph/pull/18171#discussion_r144685544

Reply:
@tchaikov  I messed that up, I have changed it now.